Rubberduck.Parsing/VBA/ParseCoordinator.cs

```diff
@@ -839,22 +839,12 @@ private void SyncComReferences(IReadOnlyList<IVBProject> projects, CancellationT
             var unmapped = new ConcurrentBag<IReference>();
 
             var referencesToLoad = GetReferencesToLoadAndSaveReferencePriority(projects);
-                            
-            State.OnStatusMessageUpdate(ParserState.LoadingReference.ToString());
 
-            var referenceLoadingTaskScheduler = ThrottelingTaskScheduler(_maxReferenceLoadingConcurrency); 
+            State.OnStatusMessageUpdate(ParserState.LoadingReference.ToString());
```

Comment:
Shouldn't this get the localized resource for the `ParserState.LoadingReference` state? The `InspectionUI` resources will be moving to the `Rubberduck.Parsing` assembly; perhaps the `ParserState` enum resources should be moved to `ParsingText.resx`?